### PR TITLE
Fixed several issues with `exit-status.sh` script

### PR DESCRIPTION
**Pull request by alexreg**
- Returns success/failure exit code based on whether all of the individual tests passed or not.
- Added options for specifying path of `verapdf` program (`VERAPDF`) and whether to change dir to /tmp first or not (`NO_CD`).
- Made use of `readlink` more robust — note that `-f` option is not available on BSD and macOS.

I am not sure if `SCRIPT_DIR` is actually used at all... it didn't seem to be. If so, maybe it can be removed.

Anyway, I am making these changes so that this test script can be used much more easily by packages in package managers (e.g., the Homebrew formula I use, which needs to run a test script). I have kept the default behaviour exactly the same as before, except that the script itself now changes its exit status. I hope this is okay.

### CODE REVIEW

File: tests/exit-status.sh
Comment (carlwilson) [inline tests/exit-status.sh:3]: Point taken about these, the whole construct appears unused and lines 2-9 can go.

File: tests/exit-status.sh
Comment (carlwilson) [inline tests/exit-status.sh:22]: [Codacy points out that this isn't used](https://app.codacy.com/gh/veraPDF/veraPDF-apps/pullRequest?prid=8443762) and again lines 21-23 can probably be removed as well.

**Review by carlwilson — COMMENTED** [review]
@alexreg thanks for the excellent work. The review points are really existing issues that it would be nice to clean up while the PR is about. If you want to delete the salient lines that would be great. If not, just let me know here and I'll submit a follow-up PR.

Many thanks, sorry for the cheeky extra request.

File: tests/exit-status.sh
Comment (alexreg) [reply tests/exit-status.sh]: Okay, sounds good.

File: tests/exit-status.sh
Comment (alexreg) [reply tests/exit-status.sh]: Sorry, line 22 should be assigning `VERAPDF`, that was my mistake. I set this as an environment variable in the install script for the package, since it's not located at `./verapdf/verapdf` in that case. Hope that's alright.

**Review by carlwilson — APPROVED** [review]
That's great thanks @alexreg :+1:

**Comment by alexreg** [comment]
@carlwilson Thanks for reviewing. The two things above have been resolved now.

Once this is merged and a new release is made, I should be able to get my PR for the Homebrew verapdf-apps formula (package) accepted, which will be great. (Don't want to pressure you into a new release out of schedule, but if you have an estimate of when the next one will be, that would be super.)

**Comment by carlwilson** [comment]
@alexreg I can say that we are looking to get a release candidate out imminently. In truth, I'm not the one doing the work (it would probably be delayed if I was ;) ) so mine is not the final word here.

**Comment by alexreg** [comment]
@carlwilson Okay, that's good to know, thanks.

Did you catch my other reply by the way? No rush, just curious.

**Comment by carlwilson** [comment]
I've passed a bit of extra information on for now so you can see how it works, am happy to assist a little depending on the best way forward for you.

**Pull request merged**
